collect_app/src/main/java/org/odk/collect/android/adapters/FileArrayAdapter.java

```diff
@@ -103,4 +115,15 @@ public View getView(int position, View convertView, @NonNull ViewGroup parent) {
         holder.onBind(items.get(position));
         return view;
     }
+
+    private boolean isNewVersion(DriveListItem item) {
+        FormsDao formsDao = new FormsDao();
+        Cursor cursor = formsDao.getFormsCursorForFormFilePath(Collect.FORMS_PATH + File.separator + item.getName());
+        if (cursor != null && cursor.getCount() > 0) {
```

Comment:
I think it's better to use `cursor != null && cursor.moveToFirst()` like we do in other places. `cursor.moveToFirst()` will return false if `cursor.getCount() <= 0` and make sure you are at the first position.